builder/virtualbox/iso/test-fixtures/http/preseed.cfg

```diff
@@ -0,0 +1,42 @@
+# Preseeding only locale sets language, country and locale.
```

Comment:
I know packer has a lot of `test-fixtures/` folders but a good practice in go is to put test files in the `testdata` folder; so that the go compiler ignores these folders.

Some explanations here: https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
should I just rename test-fixtures to testdata? Is this enough?